.github/scripts/shared/utils.ts

```diff
@@ -39,12 +39,34 @@ export function getCurrentDateFormatted(): string {
 
 // This mapping is used to know what planning repo is used for each code repo
 export const codeRepoToPlanningRepo: { [key: string]: string } = {
-  "metamask-extension": "MetaMask-planning",
-  "metamask-mobile": "mobile-planning"
-}
+  'metamask-extension': 'MetaMask-planning',
+  'metamask-mobile': 'mobile-planning',
+};
 
 // This mapping is used to know what platform each code repo is used for
 export const codeRepoToPlatform: { [key: string]: string } = {
-  "metamask-extension": "extension",
-  "metamask-mobile": "mobile",
+  'metamask-extension': 'extension',
+  'metamask-mobile': 'mobile',
+};
+
+export const sleep = (ms: number) =>
+  new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
`const { setTimeout: sleep} = require('node:timers/promises');` is a thing you can use in Node.js :-)